CMGTools/TTHAnalysis/python/analyzers/treeProducerSusyMultilepton.py

```diff
@@ -126,4 +127,9 @@
             "ivf"       : NTupleCollection("SV",     svType, 20, help="SVs from IVF"),
             "genBHadrons"  : NTupleCollection("GenBHad", heavyFlavourHadronType, 20, mcOnly=True, help="Gen-level B hadrons"),
             "genDHadrons"  : NTupleCollection("GenDHad", heavyFlavourHadronType, 20, mcOnly=True, help="Gen-level D hadrons"),
+            ##------------------------------------------------
+            "jetsNonTauIdx" : NTupleCollection("JetNoTauIdx",objectInt, 10, help="index of jets not associated to taus"),
+            "bjetsMNonTauIdx" : NTupleCollection("BJetMediumNoTauIdx",objectInt, 10, help="index of b-jets (CSV medium) not associated to taus"),
```

Comment:
Why do you need `bjetsMNonTauIdx` ? we're not saving the colleciton of b-jets so this is probably unnecessary.